app/assets/javascripts/components/students_grid.es6.jsx

```diff
@@ -1,19 +1,48 @@
 class StudentsGrid extends Component {
 
+  static get propTypes() {
+    return {
+      students: React.PropTypes.array.isRequired,
+    };
+  }
+
+  static get defaultProps() {
+    return {
+      students: [],
+    };
+  }
+
   get styles() {
     return {
       container: {
         display: 'flex',
+        flexWrap: 'wrap',
+        justifyContent:'space-between',
+        alignContent: 'flex-start',
+        flex: '1',
         position: 'relative',
-        marginTop: '2%',
       },
     };
   }
 
+  renderCard(student) {
+    return (
+      <StudentsCard
+        key={student.id}
+        student={student} />
+    );
+  }
+
+  renderCards() {
+    return (
+      this.props.students.map(this.renderCard.bind(this))
+    );
+  }
```

Comment:
Since this is just a regular `return` (not a 'jsx'), we don't need the extra parentheses.
Change
```
return (
  this.props.students.map(this.renderCard.bind(this))
);
```
to
`return this.props.students.map(this.renderCard.bind(this));`.